<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>176</width>
          <height>319</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="1" colspan="2">
          <widget class="QSpinBox" name="originY_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Rojo:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Destino Y:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QSpinBox" name="red_spinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Verde:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QSpinBox" name="destX_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QSpinBox" name="originX_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QSpinBox" name="blue_spinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Velocidad:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="id_lineEdit"/>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="QSpinBox" name="green_spinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QPushButton" name="addEnd_pushButton">
           <property name="text">
            <string>Agregar Final</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Azul:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="originX_label_5">
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QSpinBox" name="destY_spinBox">
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="originX_label_6">
           <property name="text">
            <string>Origen X:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="3">
          <widget class="QPushButton" name="showListParticle_pushButton">
           <property name="text">
            <string>MOSTRAR</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Destino X:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="originY_label_3">
           <property name="text">
            <string>Origen Y:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QPushButton" name="addToStart_pushButton">
           <property name="text">
            <string>Agregar Inicio</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QSpinBox" name="speed_spinBox">
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPlainTextEdit" name="particle_PlainText">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>181</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="idSort_pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="distanceSort_pushButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>DISTANCIA</string>
        </property>
       </widget>
       <widget class="QPushButton" name="speedSort_pushButton">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>VELOCIDAD</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>313</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ORDENAR PARTICULAS</string>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>0</y>
          <width>621</width>
          <height>331</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Table">
         <attribute name="title">
          <string>Tabla</string>
         </attribute>
         <widget class="QTableWidget" name="particle_tableWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>541</width>
            <height>291</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="search_lineEdit">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>10</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="search_pushButton">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>101</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
         <widget class="QPushButton" name="show_pushButton">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>10</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Mostrar Todo</string>
          </property>
         </widget>
         <widget class="QLabel" name="originX_label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Dibujar</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="clearDraw_pushButton">
            <property name="text">
             <string>LIMPIAR</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="only_particles_pushButton">
            <property name="text">
             <string>SOLO PARTICULAS</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="draw_pushButton">
            <property name="text">
             <string>DIBUJAR</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="close_particle_pushButton">
            <property name="text">
             <string>PARTICULAS MAS CERCANAS</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbrir">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuAbrir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
